fonts.css

```diff
@@ -0,0 +1,22 @@
+@font-face {
+	font-family: 'MuseoSansCyrl';
+	font-display: swap;
+	src: url("../fonts/MuseoSansCyrl.eot");
+	src: local("O"), url("../fonts/MuseoSansCyrl.woff") format("woff"), 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/gh/WebHeroSchool/baliproject/pullRequest?prid=5558708)